plugins/defaultplugins/common/model/srv6/srv6.proto

```diff
@@ -0,0 +1,95 @@
+syntax = "proto3";
+
+package srv6;
+
+message LocalSID {          /* SID(IPv6 Address) is part of key and NOT part of value(message) in ETCD key-value store */
+    uint32 fibTableID = 1;  /* ID of FIB table where to install localsid entry */
+
+    /* Configuration for end functions (all end functions are mutually exclusive) */
+    End baseEndFunction = 2;
+    EndX endFunctionX = 3;
+    EndT endFunctionT = 4;
+    EndDX2 endFunctionDX2 = 5;
+    EndDX4 endFunctionDX4 = 6;
+    EndDX6 endFunctionDX6 = 7;
+    EndDT4 endFunctionDT4 = 8;
+    EndDT6 endFunctionDT6 = 9;
+
+    /* End function behavior of simple endpoint */
+    message End {
+        bool psp = 1;                   /* use PSP (penultimate segment POP of the SRH) or by default use USP (Ultimate Segment Pop of the SRH) */
+    }
+
+    /* End function behavior of endpoint with Layer-3 cross-connect*/
+    message EndX {
+        bool psp = 1;                   /* use PSP (penultimate segment POP of the SRH) or by default use USP (Ultimate Segment Pop of the SRH) */
+        string outgoingInterface = 2;   /* name of cross-connected outhgoing interface */
+        string nextHop = 3;             /* next hop address for cross-connected link */
+    }
+
+    /* End function behavior of endpoint with specific IPv6 table lookup*/
+    message EndT {
+        bool psp = 1;                   /* use PSP (penultimate segment POP of the SRH) or by default use USP (Ultimate Segment Pop of the SRH) */
+    }
+
+    /* End function nehavior of endpoint with decapulation and Layer-2 cross-connect (or DX2 with egress VLAN rewrite when VLAN notzero - not supported this variant yet)*/
```

Comment:
Typo: `decapulation`.

Reply:
fixed in 86d29e635ee351c4a0ccbfde26b1baa67a7c95ce